Gruntfile.js

```diff
@@ -0,0 +1,71 @@
+module.exports = function (grunt) {
+
+  grunt.loadNpmTasks('grunt-babel');
+  grunt.loadNpmTasks('grunt-contrib-clean');
+  grunt.loadNpmTasks('grunt-contrib-uglify');
+  grunt.loadNpmTasks('grunt-banner');
+
+  var LICENSE = '/****\n * ' +
+    grunt.file.read('./LICENSE.md', {encoding: 'utf-8'}).replace(/\n/ig, '\n * ') +
+    '\n ****/';
+
+  grunt.initConfig({
+    babel: {
+      options: {
+        ast: false,
+        sourceMaps: false,
+        nonStandard: false,
+        compact: 'false',
+        modules: 'umd',
+        moduleId: 'Timsort',
```

Comment:
Another annoying and not-so-useful comment. Is there a reason for the moduleId to be `Timsort` rather than `timsort`? If not, can we switch to `timsort` (just for consistency with the module name).